package.json

```diff
@@ -178,7 +178,13 @@
         {
           "int32Hint": false
         }
-      ]
+      ],
+      "no-restricted-imports": ["warn", {
```

Comment:
How about error? Will it break current code-base?

Reply:
Now this library is still used in the project. Therefore, I think a warning will suffice.